src/diffusers/models/modeling_utils.py

```diff
@@ -150,7 +151,23 @@ def load(module: torch.nn.Module, prefix=""):
     return error_msgs
 
 
-class ModelMixin(torch.nn.Module):
+# Taken from
+# https://github.com/huggingface/transformers/blob/5bb4430edc7df9f9950d412d98bbe505cc4d328b/src/transformers/modeling_utils.py#L3939C1-L3950C21
+def unwrap_model(model: nn.Module) -> nn.Module:
```

Comment:
Where do we use this function?

Reply:
https://github.com/huggingface/diffusers/pull/4128/files#diff-6a3b9a08c1d37dbc341131632415fea800af242a84fb31f1bcd40d725e2eeeebR340